pages/events/_event.vue

```diff
@@ -53,14 +53,17 @@
       <v-divider></v-divider>
       <h1 class="text-xs-center primary--text font-lato mt-3">Sponsors</h1>
       <v-container pa-4>
-        <v-layout row wrap justify-space-around>
-          <v-flex xs12 sm3 v-for="sponsor in currentEvent.sponsors" :key="sponsor.name">
-            <div class="vv-day-sponsor px-4 mb-3">
-              <a :href="sponsor.website" target="_blank">
-                <img :src="sponsor.img" class="vv-day-sponsor-logo">
-              </a>
-            </div>
-          </v-flex>
+        <v-layout column v-for="category in sponsors" :key="category.name">
```

Comment:
I don't think we're going to have sponsors categories for events, it seems way too detailed for me.
@jlooper What do you think?

Reply:
hi, I believe that having sponsor categories for events will be hard to standardize across all the events thrown by all our chapters. What I recommend is, if you have a sponsor category for your event, put that information in the text of your sponsor description like:

Angel Bread Company
Gold Sponsor for Food 
Angel provides us with delicious brioches for our events. Learn more about them at sponsorwebsite.com